databallpy/data_parsers/event_data_parsers/statsbomb_parser.py

```diff
@@ -0,0 +1,507 @@
+        "Shot": "shot",
+        "Dribble": "dribble",
+    }
+
+    shot_events = {}
+    pass_events = {}
+    dribble_events = {}
+    
+    event_mask = [event["type"]["id"] not in [5, 18, 19, 26, 27, 34, 35, 36] for event in events_json]
+    for id, event in enumerate(np.array(events_json)[event_mask]): 
+        event_data["event_id"].append(id)
+        event_data["period_id"].append(event["period"])
+        event_data["minutes"].append(event["minute"])
+        event_data["seconds"].append(event["second"])
+        event_data["player_id"].append(event["player"]["id"])
+        event_data["team_id"].append(event["possession_team"]["id"])
+        if "location" in event.keys():
+            event_data["start_x"].append(event["location"][0]-(pitch_dimensions[0]/2))
+            event_data["start_y"].append(event["location"][1]-(pitch_dimensions[1]/2))
+        else:
+            event_data["start_x"].append(MISSING_INT)
+            event_data["start_y"].append(MISSING_INT)
+        event_data["datetime"].append(
+            pd.to_datetime(metadata.periods_frames["start_datetime_ed"][event["period"]-1]+pd.to_timedelta(event["minute"]*60+event["second"], unit="seconds"))
+        )
+        event_data["statsbomb_event"].append(event["type"]["name"].lower().replace("*",""))
+        event_data["statsbomb_event_id"].append(event["id"])
+        event_data["player_name"].append(event["player"]["name"])
+        event_data["team_name"].append(event["possession_team"]["name"])
+
+        event_type_object = event["type"]["name"].lower().replace(" ", "_").replace("*", "").replace("-", "_").replace("/", "-")
+
+        if event["type"]["name"] in databallpy_mapping:
+            databallpy_event = databallpy_mapping[event["type"]["name"]]
+            event_data["databallpy_event"].append(databallpy_event)
+            try:
+                event_data["outcome"].append(event[event_type_object]["outcome"]["name"])
+            except:
+                event_data["outcome"].append("")
+            if databallpy_event == "shot": 
+                shot_events[id] = _get_shot_event(event=event, id=event["id"], pitch_dimensions=pitch_dimensions, periods=metadata.periods_frames, away_team_id=metadata.away_team_id)
+            elif databallpy_event == "pass":
+                pass_events[id] = _get_pass_event(event=event, id=event["id"], pitch_dimensions=pitch_dimensions, periods=metadata.periods_frames, away_team_id=metadata.away_team_id)
+            elif databallpy_event == "dribble":
+                dribble_events[id] = _get_dribble_event(event=event, id=event["id"], pitch_dimensions=pitch_dimensions, periods=metadata.periods_frames, away_team_id=metadata.away_team_id)
+        else:
+            event_data["databallpy_event"].append(None)
+
+            try:
+                event_data["outcome"].append(event[event_type_object[event["type"]["id"]]]["outcome"]["name"])
+            except:
+                event_data["outcome"].append("")
```

Comment:
Seems like duplicate code from the "outcome above", can we put this somewhere outside of the `databallpy_mapping` if statement to capture it for all in events at once? Or is this inconvenient?